src/Static.js

```diff
@@ -28,11 +26,18 @@ class Static extends React.Component {
       errorDescription: 'Something went wrong when importing your custom setting file. Please try again or proceed with default settings.',
       signInTitle: this.props.signInTitle,
       signInStatus: this.props.signInStatus,
-      importSettingsTitle: this.props.importSettingsTitle,
+      loadingTime: 'Total loading time: ',
```

Comment:
Do we need to use `this.props.` here or?

Reply:
No need for that since we are using `this.state`